website_form_builder/static/src/js/snippets.js

```diff
@@ -0,0 +1,460 @@
+/* Copyright 2017 Tecnativa - Jairo Llopis
+ * License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). */
+
+odoo.define('website_form_builder.snippets', function(require){
+    "use strict";
+
+    var base = require('web_editor.base');
+    var core = require('web.core');
+    var Model = require('web.Model');
+    var options = require('web_editor.snippets.options');
+    var _t = core._t;
+    var widgets = require("website_form_builder.widgets");
+    var data = require("web.data");
+
+    core.qweb.add_template(
+        "/website_form_builder/static/src/xml/snippets.xml"
+    );
+
+    var _fields_asked = {},
```

Comment:
IMO we should use JS naming conventions & camelCase the vars/methods

Reply:
I know those common JS conventions, but in Odoo it seems they decided to match the python ones. Just give a glance at https://www.odoo.com/documentation/11.0/reference/javascript.html and you'll see. So, I prefer to stick with those.